decidim-admin/app/commands/decidim/admin/update_participatory_process.rb

```diff
@@ -21,17 +21,25 @@ def initialize(participatory_process, form)
       # Returns nothing.
       def call
         return broadcast(:invalid) if form.invalid?
-
         update_participatory_process
-        broadcast(:ok)
+
+        if @participatory_process.valid?
+          broadcast(:ok, @participatory_process)
+        else
+          form.errors.add(:hero_image, @participatory_process.errors[:hero_image]) if @participatory_process.errors.include? :hero_image
```

Comment:
It's ok for now, but I think we should create a set of validators that can inspect a file and act at the form level instead of the model.